torch/ao/quantization/observer.py

```diff
@@ -1850,6 +1852,79 @@ def calculate_qparams(self) -> tuple[torch.Tensor, torch.Tensor]:
         and returns a tuple of scale and zero_point Tensor
         """
 
+    def convert(self, model: torch.fx.GraphModule, observer_node: Node):
```

Comment:
is this implementation the same as other existing convert? do we just need to keep one copy of the code?